pkgs/top-level/aliases.nix

```
@@ -1121,6 +1121,7 @@ mapAliases ({
   yacc = bison; # moved from top-level 2021-03-14
   yafaray-core = libyafaray; # Added 2022-09-23
   yarn2nix-moretea-openssl_1_1 = throw "'yarn2nix-moretea-openssl_1_1' has been removed."; # Added 2023-02-04
+  yubikey-manager4 = throw "yubikey-manager4 has been removed as yubikey-manager-qt updated to yubikey-manager(5). Please try to update, too."; # Added 2024-01-14
```

```suggestion
  yubikey-manager4 = throw "yubikey-manager4 has been removed, since it is no longer supported by yubikey-manager-qt. Please update to yubikey-manager."; # Added 2024-01-14
```

It's preferrable to not reference new versions, it can confuse people.